ingress-controller/internal/controllers/configuration/secret_controller.go

```diff
@@ -42,8 +43,8 @@ type CoreV1SecretReconciler struct {
 	DataplaneClient  controllers.DataPlane
 	CacheSyncTimeout time.Duration
 
-	ReferenceIndexers ctrlref.CacheIndexers
-	LabelSelector     string
+	ReferenceIndexers        ctrlref.CacheIndexers
+	LabelSelectorMatchLabels map[string]string
```

Comment:
REVIEW: Should we remove the label selector in controllers for the dedicated types as we have the manager level label selectors?

Reply:
I wonder if watch predicates actually work with label selectors set at the manager level 🤔 

If they do I think we can remove them here (ideally making a note that we "inherit" the label selectors from the manager).

Reply:
Removed them.